README.md

```diff
@@ -144,11 +144,14 @@ them to fully utilize allocated notes. You thus do not have to manually aggregat
 
     * Manual request in PBS
 
-      ``$ qsub <your-params-of-qsub> -- hq worker start``
+      ``$ qsub <your-params-of-qsub> -- `which pbsdsh` hq worker start``
+
+      > Note: `pbsdsh` makes sure that a worker will be started on each allocated
```

Comment:
Maybe also include the original one if you have only one node and have a problem with pbsdsh?